### DISCUSSION TITLE
High CPU usage with only 3 instruments

### DISCUSSION THREAD

**Discussion by bukru** [SynthstromAudible/DelugeFirmware#2666] [Q&A]
I have a track with only 
- 1 multisample piano playing chords (3 notes)
- 1 multisample piano playing mélodie (no chord)
- 1 stock drum kit playing simple rythme (kick, snare, hithat)

=> The play button is sometime flashing
=> If i try to add an other synth i try to change synth kit i have Big lag 

Is that normal ? I tried to reboot, same stuff

**Comment by bukru** [comment]
I'm on bethowen

**Comment by seangoodvibes** [comment]
> I have a track with only
> 
> * 1 multisample piano playing chords (3 notes)
> * 1 multisample piano playing mélodie (no chord)
> * 1 stock drum kit playing simple rythme (kick, snare, hithat)
> 
> => The play button is sometime flashing => If i try to add an other synth i try to change synth kit i have Big lag
> 
> Is that normal ? I tried to reboot, same stuff

multi samples are notoriously bad on the cpu because they are sample based

the deluge has a greater cpu bandwidth for synth voices than it does for samples

what is the release time on your synths? long release can result in high cpu usage
are you using a lot of automation on those synth clips?
how big are the multi samples? if they are a lot of big files loaded into memory that can slow things down

**Reply by bukru** [reply]
- Realease time => 16
- No automation
- There is two folders, one with low velocity sounds loaded to OSC 1, one with high velocity sounds loaded to OSC 2, each with OSC volume to 25  with velocity modulation  -50 for osc1 +50 to osc2. **Total size 575Mo**

As i had to copy the first piano to an other synth kit to use it at the same time, does it load 575Mo two times in memory ? Does it exist a solution to use an instrument twice without loading it two times in memory ? Or have you an advice for me ?

Thanks :+1:
